requirements.txt

```diff
@@ -1,15 +1,21 @@
-botocore==1.8.9
-Cython
-dask==0.16.1
+botocore==1.10.16
+dask==0.17.4
 docutils
+fakeredis
 future
 h5py
+ipaddress
 jmespath==0.9.3
 katversion
+netifaces
 numpy
-pkginfo
 pyephem
 python-dateutil
+python-lzf==0.2.4
+rdbtools==0.1.12
+redis
 six
 toolz==0.9.0
-git+ssh://git@github.com/ska-sa/katpoint#egg=katpoint
+
+git+ssh://git@github.com/ska-sa/katpoint
+git+ssh://git@github.com/ska-sa/katsdptelstate
```

Comment:
I'm never quite clear on when pip needs the egg= - do you know?

Reply:
I'm trying to figure this out now...

This feature seems to have been more useful in the `easy_install` days. Its only relevant purpose seems to be the following (quoting @carljm from [here](https://groups.google.com/forum/#!topic/python-virtualenv/-2CZMRFixJg)):

```
In pip AFAIK the main purpose of the #egg= fragment is to help pip sort out dependencies; so if you have another package that depends on a package you're installing editable (or from a direct URL), using the fragment allows pip to know earlier in the process that you've got that dependency covered and it doesn't need to go finding it. 
```

So I guess I'll put it back.